12-hooks-in-react/src/App.js

```diff
@@ -16,10 +17,13 @@ const items = [
   },
 ];
 
-export default () => {
+const App = () => {
   return (
     <div>
-      <Accordion items={items} />
+      {/* <Accordion items={items} /> */}
```

Comment:
Remove unused commented out code

Reply:
This code will be implemented in the future, hence commented out for now and will return at a later date.

Reply:
Since we are practicing PR, you are now currently merging code to the master branch which in this way is unacceptable, it's just one line, if you feel the need to readd, just remember or reference this PR in the future.

Reply:
Noted on this, thanks!